compiler/passes/src/loop_unrolling/statement.rs

```diff
@@ -50,39 +50,39 @@ impl StatementReconstructor for UnrollingVisitor<'_> {
             return (Statement::Iteration(Box::new(input)), Default::default());
         };
 
-        input.start_value.replace(Some(Value::try_from(start_lit).unwrap()));
-        input.stop_value.replace(Some(Value::try_from(stop_lit).unwrap()));
+        let start_value = Value::try_from(start_lit).unwrap();
```

Comment:
Not directly related to this PR, but would you be able to add a comment on why the unwrap is safe.